<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADRES_ETIKETLERI_BATCH</class>
 <widget class="QDialog" name="ADRES_ETIKETLERI_BATCH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>337</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>337</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>337</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Etiket Yazdır</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0" columnstretch="50,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>212</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_batch_button" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Sayfa Düzeni</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,6,10,1,1,1,1,1,1,1" columnstretch="1,5,1,2,1,5,1,2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Etiket Şablonları </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QComboBox" name="combo_box_sablonlar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Etiket Yüksekliği                   </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCommaEdit" name="comma_edit_etiket_yukseligi">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Etiket Genişligi </string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QCommaEdit" name="comma_edit_etiket_genisligi">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="label_19">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Etiketler Arasındaki Boşluk  </string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCommaEdit" name="comma_edit_etiketler_arasi_bosluk">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_13">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="label_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Yazı Tipi Boyutu </string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QLineEdit" name="line_edit_yazi_tipi_boyutu">
          <property name="text">
           <string>11</string>
          </property>
         </widget>
        </item>
        <item row="4" column="6">
         <widget class="QLabel" name="label_17">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>pt</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Satırlar Arasındaki Boşluk    </string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCommaEdit" name="comma_edit_satirlar_arasi_bosluk">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_14">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Sayfa Üst Boşluk                 </string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCommaEdit" name="comma_edit_ust_bosluk">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_15">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Sayfa Sol Boşluk                  </string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCommaEdit" name="comma_edit_sol_bosluk">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_16">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Sayfadaki Satır Sayısı          </string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QCommaEdit" name="comma_edit_satir_sayisi">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Satırdaki Etiket Sayısı         </string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QCommaEdit" name="comma_edit_satirdaki_etiket_sayisi">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="7">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="6" colspan="2">
         <widget class="QPushButton" name="push_button_etiket">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCommaEdit</class>
   <extends>QLineEdit</extends>
   <header>QCommaEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>combo_box_sablonlar</tabstop>
  <tabstop>push_button_etiket</tabstop>
  <tabstop>comma_edit_etiket_yukseligi</tabstop>
  <tabstop>comma_edit_etiket_genisligi</tabstop>
  <tabstop>comma_edit_etiketler_arasi_bosluk</tabstop>
  <tabstop>line_edit_yazi_tipi_boyutu</tabstop>
  <tabstop>comma_edit_satirlar_arasi_bosluk</tabstop>
  <tabstop>comma_edit_ust_bosluk</tabstop>
  <tabstop>comma_edit_sol_bosluk</tabstop>
  <tabstop>comma_edit_satir_sayisi</tabstop>
  <tabstop>comma_edit_satirdaki_etiket_sayisi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
